src/mixer/basetrackplayer.h

```diff
@@ -41,6 +42,8 @@ class BaseTrackPlayer : public BasePlayer {
     void loadTrackFailed(TrackPointer pTrack);
     void newTrackLoaded(TrackPointer pLoadedTrack);
     void unloadingTrack(TrackPointer pAboutToBeUnloaded);
+    void noPassthroughInputConfigured();
+    void noVinylControlInputConfigured();
 };
 
 class BaseTrackPlayerImpl : public BaseTrackPlayer {
```

Comment:
The "Base" prefix and "Impl" suffix is somehow unusual. I have not yet an overview about the object hierarchy, but it looks like we can improve the naming here to clarify it.